Packs/SpiderSilk/Integrations/SpiderSilk/SpiderSilk.yml

```diff
@@ -0,0 +1,154 @@
+  - arguments: []
+    description: Fetches dark web credential exposures as incidents from the /darkweb_credentials endpoint.
+    name: fetch-incidents
+  - arguments:
+    - description: Number of hours since reports have been created (e.g., 0.5 to 168).
+      name: reported_since_hours
+    - defaultValue: '50'
+      description: Maximum number of credentials to return per page.
+      name: limit
+    - defaultValue: '1'
+      description: Page number for pagination.
+      name: page
+    description: Retrieve the information of the Darkweb credentials associated to the Company profile.
+    name: spidersilk-get-darkweb-credentials
+  - arguments:
+    - defaultValue: '50'
+      name: limit
+    - defaultValue: '1'
+      name: page
+    description: Retrieve the information of the Darkweb reports associated to the Company profile.
+    name: spidersilk-get-darkweb-reports
+  - arguments:
+    - description: The UUID of the darkweb report.
+      name: uuid
+      required: true
+    description: Retrieve the information of the darkweb by its uuid.
+    name: spidersilk-get-darkweb-report-details
+  - arguments:
+    - description: The UUID of the darkweb report to update.
+      name: uuid
+      required: true
+    - description: The integer ID of the new status.
+      name: status_id
+      required: true
+    - description: An optional comment for the status change.
+      name: comment
+    description: Change status of the darkweb.
+    name: spidersilk-update-darkweb-report-status
+  - arguments:
+    - defaultValue: '50'
+      name: limit
+    - defaultValue: '1'
+      name: page
+    - description: Number of hours since threats were updated.
+      name: updated_since
+    description: Retrieve the information of the threats associated to the Company profile.
+    name: spidersilk-get-threats
+  - arguments:
+    - description: The UUID of the threat.
+      name: uuid
+      required: true
+    description: Retrieve the information of the threat by its uuid.
+    name: spidersilk-get-threat-details
+  - arguments:
+    - description: The UUID of the threat to update.
+      name: uuid
+      required: true
+    - description: The integer ID of the new status.
+      name: status_id
+      required: true
+    - description: An optional comment for the status change.
+      name: comment
+    description: Change status of the threat.
+    name: spidersilk-update-threat-status
+  - arguments:
+    - defaultValue: '50'
+      name: limit
+    - defaultValue: '1'
+      name: page
+    description: Retrieve the information of the assets associated to the Company profile.
+    name: spidersilk-get-assets
+  - arguments:
+    - description: The ID of the asset (e.g., 10b95cb930bf4bc79bfdf21173e7781f).
+      name: asset_id
+      required: true
+    description: Retrieve the information of the asset by its Id.
+    name: spidersilk-get-asset-details
+  dockerimage: xsoar-registry.pan.dev/demisto/python3:3.10.13.83255
```

Comment:
the docker image entry is not correct, i assume you meant to put demisto/python3:3.10.13.83255
but even with that, this image is old, the latest image is "demisto/python3:3.12.11.4508456", please update it